### PR TITLE
added main function and added to git ignore

### PR DESCRIPTION
**Pull request by logan-anderson**

### CODE REVIEW

File: Downloader.py [hunk 1]
```diff
@@ -92,17 +92,18 @@ def download_cezar():
...
-
-config_file = open('config.json', 'w')
-pretty_json = json.dumps(config,
-    indent=4, 
-    separators=(',', ': ')
-    )
-config_file.write(pretty_json)
-config_file.close()
+if __name__== "__main__":
+    print('test')
```
Comment (logan-anderson) [inline Downloader.py]: This is not good

Comment (Alexander-Cairns) [inline Downloader.py]: https://www.youtube.com/watch?v=Ulu90E_QGmc

**Review by Alexander-Cairns — CHANGES_REQUESTED** [review]
Look at my comment

**Review by Alexander-Cairns — APPROVED** [review]
Looks good. 
My name is Alexander and I approve this request.

**Pull request merged**